src/odemis/util/test/transform_test.py

```diff
@@ -348,39 +354,29 @@ def test_to_physical_space_known_values(self):
 
         """
         # tuple
-        for ji, xy, xy2, xy23 in zip(self._ji, self._xy, self._xy2, self._xy23):
-            res = to_physical_space(ji, self._shape)
-            res2 = to_physical_space(ji, self._shape, pixel_size=2)
-            res23 = to_physical_space(ji, self._shape, pixel_size=(2, 3))
-
-            numpy.testing.assert_array_almost_equal(xy, res)
-            numpy.testing.assert_array_almost_equal(xy2, res2)
-            numpy.testing.assert_array_almost_equal(xy23, res23)
+        for shape, pixel_size, _xy in self._known_values:
+            for ji, xy in zip(self._ji, _xy):
+                res = to_physical_space(ji, shape, pixel_size)
+                numpy.testing.assert_array_almost_equal(xy, res)
```

Comment:
Double for loop is unnecessary, `to_physical_space` can take an array as input. (same in other places)

``` python
for shape, pixel_size, _xy in self._known_values:
    res = to_physical_space(self._ji, shape, pixel_size)
    numpy.testing.assert_array_almost_equal(_xy, res)
```

Reply:
The goal here is to test with a tuple as input; hence the double for loop